README.md

```diff
@@ -10,30 +13,41 @@ Join the community on the [ContinualAI slack](https://join.slack.com/t/continual
 - Install Unix command line tools (like `rm`, and `make`) 
     - On Linux, you already have everything
     - On Mac, install `homebrew` and through homebrew, the `make` package
-    - On Windows... idk, wipe your hard drive and install a proper OS? (Or try Cygwin or Windows Subsystem for Linux)
+    - On Windows, not 100% sure, but try using Powershell, Cygwin or Windows Subsystem for Linux
 - Install Miniconda/Anaconda and have the conda/python/pip binaries for that available on your `$PATH` and in your shell.
 
 
 ## Setup
 
-1. Get your API Authentication token from the [EvalAI Website in your profile](https://eval.ai/web/profile).
+1. Get your API Authentication token from [you profile on the EvalAI Website](https://eval.ai/web/profile),
+   and copy it into the `evalai_token.txt` file.
 
-2. From the root directory of the repository, run the following command:
+2. From the root directory of the repository, run the `./install.sh` script and activate the `clcomp21` environment.
 
 ```console
-./install.sh <EVALAI_TOKEN>
+./install.sh
```

Comment:
TBH I don't like the token to text file thingy. You only need to run this once so why go through the trouble of creating a text file, pasting there, and saving the text file, when the first and last step of that sequence are entirely unnecessary and you could just paste it here? The evalai token file is never reused for anything and internally it stores the token somewhere else anyway.

Reply:
My thinking is that the install script can be used to update the dependencies if need be later down the line, in which case it really is a pain to have to go fetch your token every time.
It's not a big deal here, but it's generally not good practice to keep tokens in the command history.

Even just for mewhen debugging the installation instructions, it's actually a bit of a pain to have to go back and forth every time I want to create the env / update the packages

Reply:
I'm sorry, but what? Both leave the exact same thing in your file system - your command history is stored in a human-readable file and so is what you're suggesting here. And no, the install script should not be run multiple times. We can make a `make upgrade` command for that, but the install script should only be run 1 single time.